src/_includes/components/testimonials-slider.html

```diff
@@ -1,8 +1,8 @@
 <div class="c-slider c-slider--testimonials" data-slider data-aria-label="Testimonials">
     <div class="c-slider__slides-container" data-slides>
-        <div class="c-slider__slides-wrapper" data-slides-wrapper>
+        <div class="c-slider__slides-wrapper" tabindex="-1" data-slides-wrapper>
```

Comment:
@kecrily based on @SaraSoueidan's comments, can we revert these changes in the slider?